scripting/get5/stats.sp

```diff
@@ -227,8 +227,16 @@ public Action Stats_PlayerDeathEvent(Event event, const char[] name, bool dontBr
       IncrementPlayerStat(attacker, STAT_KILLS);
       g_PlayerRoundKillOrAssistOrTradedDeath[attacker] = true;
 
-      if (headshot)
+      if (headshot) {
         IncrementPlayerStat(attacker, STAT_HEADSHOT_KILLS);
+      }
+
+      // Because custom knives all have different names, we simply search the weapon string.
+      // StrContains returns the index and -1 if the string was not found.
+      // As all knives must *start with* 'knife', we can be sure that this must return 0 for all knives.
+      if (StrContains(weapon, "knife", true) == 0) {
```

Comment:
So it turns out that all custom knives have different string values as well, so I replaced this with a `StrContains()`. I am going to assume that all knives start with `knife`, as I have limited knives available and that was the case for all of them. This does makes sense as it indicates to the game which icon to show in the kill-feed, I just never thought of that.